### ISSUE TITLE
[a11y]: dropdown box A11y (Accessbility) violation

### ISSUE THREAD

**Issue by ivanybma** [carbon-design-system/carbon#18855]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

v1.67

### React version

18.2

### Automated testing tool and ruleset

IBM Accessibility Equal Access Toolkit: Accessibility Checker

### Description

Hi Carbon team,

As part of an IBM-wide requirement, a requirement is for DataStage UI (DataStage/ds-canvas) to be accessibility compliant, via the [IBM Equal Access Accessibility Checker tool](https://chromewebstore.google.com/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp?hl=en-US).

Screenshot:

<img width="1861" alt="Image" src="https://github.com/user-attachments/assets/a8f02de5-5511-43bb-917b-41ef916f8ad9" />

### Reproduction/example

@carbon/react

### Steps to reproduce

Create a ETL Flow using transformer node using https://www.ibm.com/products/datastage using chrome browser
For example: RowGen node -> Transformer Node -> Peek node
Add 2 column on Rowgen
Double Click on transformer node
If new transformer toggle button is not on, turn it on as it is highlighted below and open the left side panel
click the find function on the tool bar left most icon "magnifier", left side panel show up

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by alisonjoseph** [comment]
Hi, thanks for reporting this, do you mind recreating in a [Stackblitz](https://stackblitz.com/github/carbon-design-system/sandboxes/tree/main/react/latest/react-19-vite/?preset=node=) as we're not seeing this issue in Storybook.

**Comment by 2nikhiltom** [comment]
This looks like a duplicate of https://github.com/carbon-design-system/carbon/issues/17567 and https://github.com/carbon-design-system/carbon/issues/18858 .
The issue is not re-created on the latest versions (v11.78.0)

**Comment by tay1orjones** [comment]
Duplicate of https://github.com/carbon-design-system/carbon/issues/17582

**Comment by CCCCChloe** [comment]
Hi @alisonjoseph I'm seeing the same issue and created a Stackblitz project here https://stackblitz.com/edit/github-7bh8iprp?file=src%2FApp.jsx

![Image](https://github.com/user-attachments/assets/733869c0-f1de-4cf1-b605-15c5529ee119)

@2nikhiltom Could you reopen this issue thanks!
